content/en/docs/tasks/traffic-management/ingress/ingress-control/index.md

```diff
@@ -97,6 +97,12 @@ Setting the ingress IP depends on the cluster provider:
     $ gcloud compute firewall-rules create allow-gateway-https --allow "tcp:$SECURE_INGRESS_PORT"
     {{< /text >}}
 
+1.  _IBM Cloud Kubernetes Service:_
+
+    {{< text bash >}}
+    $ ibmcloud ks workers --cluster cluster-name-or-id
+    $ export INGRESS_HOST=public-IP-of-one-of-the-worker-nodes
+    {{< /text >}}
```

Comment:
```suggestion
    {{< /text >}}

```

Reply:
Oops, my mistake. I meant to leave a blank line, just remove the extra space characters. The linter is very fussy about these things :)

Reply:
```
 ./en/docs/tasks/traffic-management/ingress/ingress-control/index.md:105: MD031 Fenced code blocks should be surrounded by blank lines
```